pkg/cmd/pulumi/packagecmd/project_sdk_link.go

```diff
@@ -451,6 +451,7 @@ func linkGoPackage(root string, pkg *schema.Package, out string) error {
 	}
 
 	fmt.Printf("Go mod file updated to use local sdk for %s\n", pkg.Name)
+	fmt.Printf("To use this package, import %s\n", goInfo.ImportBasePath)
```

Comment:
n.b. ImportBasePath might not be set, so this should probably hook into gocodegen to get the default ImportBasePath as well

Reply:
what are the conditions where it is not set?

Reply:
By default.... I think the tfbridge always sets it but other provider implementations might not. 

I'm happy for you to just leave a TODO about this rather than spelunking into how go codegen sets it by default. Printing the right thing for tfbridge here is still an improvement.

Reply:
Not to stomp on due diligence here (I'll add a check), but...isn't this code _only_ used by the bridge, for the purposes of dynamic providers?

Reply:
No, this is used for any local sdk for any provider, dynamic or not.